### PR TITLE
Change to eclipse project

### PR DESCRIPTION
**Pull request by sgraband**
Signed-off-by: Simon Graband <sgraband@eclipsesource.com>

Co-authored-by: Tobias Ortmayr <tortmayr@eclipsesource.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,357 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.emfcloud.validation</groupId>
+	<artifactId>org.eclipse.emfcloud.validation.parent</artifactId>
+	<description>Model Validation Parent Pom</description>
+	<version>1.0.0-SNAPSHOT</version>
```
Comment (eneufeld) [inline pom.xml]: we should not use 1.0.0, it should rather be something like 0.2.0 or so

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+									<ws>gtk</ws>
+									<arch>x86_64</arch>
+								</environment>
+								<environment>
+									<os>macosx</os>
+									<ws>cocoa</ws>
+									<arch>x86_64</arch>
+								</environment>
+							</environments>
+							<target>
```
Comment (eneufeld) [inline pom.xml:151]: you can specify the target file directly using tycho 2.3: https://wiki.eclipse.org/Tycho/Release_Notes/2.3#Target_files_can_be_specified_directly_now

before 2.3 it is also not needed to have a classifier, the target file just needs to be named as the artifactid see https://github.com/eclipse-emfcloud/coffee-editor/tree/master/backend/releng/org.eclipse.emfcloud.coffee.target

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>p2-release</id>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+			</activation>
+			<modules>
```
Comment (eneufeld) [inline pom.xml:165]: why can't we move the modules outside the profiles? they are the same for all builds, aren't they?

File: releng/org.eclipse.emfcloud.validation.codestyle/checkstyle/model-validation-checkstyle.xml [hunk 4]
```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (eneufeld) [inline releng/org.eclipse.emfcloud.validation.codestyle/checkstyle/model-validation-checkstyle.xml]: can't we reuse the checkstyle from Tobias: https://github.com/eclipse-emfcloud/emfcloud/tree/master/codestyle

File: targetplatform/r2021-03.tpd [hunk 5]
```diff
@@ -0,0 +1,23 @@
+target "2021-03 Release" with source,requirements
+location "http://download.eclipse.org/releases/2021-03/" {
+	org.eclipse.equinox.p2.discovery.feature.feature.group
+	org.eclipse.equinox.executable.feature.group
+	org.eclipse.emf.sdk.feature.group
+}
+
+location "http://ghillairet.github.io/p2" {
```
Comment (eneufeld) [inline targetplatform/r2021-03.tpd:8]: @planger didn't we publish a version of emfjson since we moved it to eclipse?

**Review by eneufeld — CHANGES_REQUESTED** [review]
Mainly the version should be changed to something other than 1.0.0 as we won't have a release of the framework just yet. To be in eclipse incubation we need a version below 1.0.0 .
Looks great otherwise!

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+			<repositories>
+				<repository>
+					<id>sonatype</id>
+					<name>Sonatype</name>
+					<url>https://oss.sonatype.org/content/groups/public</url>
+				</repository>
+			</repositories>
+			<modules>
+				<module>org.eclipse.emfcloud.validation.framework</module>
+				<module>targetplatform</module>
```
Comment (eneufeld) [inline pom.xml]: why do we need the targetplatform in the m2 build?

**Review by eneufeld — APPROVED** [review]
Looks good. As we need to wait before we can start the cq, I suggest to wait for Philip to respond

File: targetplatform/r2021-03.tpd (hunk 5 above)
Comment (planger) [reply targetplatform/r2021-03.tpd:8]: No we didn't publish an emfjson release on the Eclipse infra yet :/
I think we'll have to stick with this for now here.

File: pom.xml (hunk 6 above)
Comment (sgraband) [reply pom.xml]: I removed it.

**Pull request merged**